modules/hardware/nvidia-jetson-orin/jetson-orin.nix

```diff
@@ -14,14 +16,64 @@
       passthrough-patch = ./pci-passthrough-agx-test.patch;
       vfio-pci = "vfio-pci.ids=10ec:c82f";
       deviceTree = "tegra234-p3701-host-passthrough.dtb";
+      buspath = [
+        {
+          bus = "pci";
+          path = "0001:01:00.0";
+        }
+      ];
+      kernelParams = [];
+      # To enable wireless with wifi uncomment SSID and password while setting enable=true
+      wireless = {
+        enable = true;
+        networks."SSID_OF_NETWORK".psk = "WPA_PASSWORD";
```

Comment:
I think this should still default to commented out - otherwise it will try to find SSID `SSID_OF_NETWORK` with password `WPA_PASSWORD`. Anyone using the wifi with passthrough to netvm on Orin AGX will have to change this line anyway.

Reply:
I'll comment these out.